src/main/java/com/redhat/red/build/koji/model/xmlrpc/messages/MultiCallResponse.java

```diff
@@ -28,8 +29,22 @@
 public class MultiCallResponse
 {
     @DataIndex( 0 )
+    private List<Object> response;
```

Comment:
The reason for `List<Object> response` instead of using `List<KojiMultiCallValueObj> valueObjs` directly is to allow the parser to convert between `KojiMultiCallFault` and `KojiMultiCallData`, but maybe there is some way to simplify this data structure.